contrib/charts/cert-manager/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: cert-manager
-version: v0.3.2
-appVersion: v0.3.0
+version: v0.4.0-dev.0
+appVersion: v0.4.0-dev.0
```

Comment:
:question: I wonder if there's any way to have this auto-generated from git, if this is being run from a git repo and git is installed. e.g.

```
$ git describe --long --tags --dirty --always
v0.3.0-59-g28c52d6f
```

Reply:
Perhaps, but for now I'd rather keep things simple. The output of `git describe ...` can/is used when linking the app (to hardcode the appversion into the binary itself). Perhaps we can explore doing this for the Helm chart too in the future, if there is some need 😄 (I'd rather not over-complicate things as part of this PR if it can be avoided!)